dvc/cachemgr.py

```diff
@@ -73,6 +74,16 @@ def __init__(self, repo):
         legacy_odb = _get_odb(repo, settings, hash_name="md5-dos2unix", **kwargs)
         self._odb["legacy"] = legacy_odb
 
+    @property
+    def fs_cache(self):
```

Comment:
Also could you please write a docstring on what this is being used for?

Reply:
Sure, added.